src/components/single-line-diagram.js

```diff
@@ -155,7 +164,7 @@ const SingleLineDiagram = forwardRef((props, ref)  => {
             });
             svgDraw.current = draw;
         }
-    }, [svg]);
+    }, [svg, updateSwitchMsg]);
```

Comment:
I don't understand why we need to put updateSwitchMsg in the dependencies here

Reply:
At first, i didn't add this dependancy, but i got strange behaviour when navigating between voltage level after an error occurred : svg not properly zoomed for example.
So i added this dependancy and it works well now ...

Reply:
Modification done